packages/typespec-autorest/test/openapi-output.test.ts

```diff
@@ -823,6 +823,52 @@ describe("identifiers decorator", () => {
     deepStrictEqual(oapi.definitions.PetList.properties.value["x-ms-identifiers"], undefined);
   });
 
+  it("ignores name/id keys for x-ms-identifiers when nested", async () => {
+    const oapi = await openApiFor(
+      `
+      @armProviderNamespace
+      @useDependency(Azure.ResourceManager.Versions.v1_0_Preview_1)
+      namespace Microsoft.Test;
+
+      model Bar {
+        foo:Foo;
+      }
+      model BarList {
+        value: Bar[]
+      }
+      model Foo
+      {
+        @key
+        name:string;
+      }
+      `,
+    );
+    deepStrictEqual(oapi.definitions.BarList.properties.value["x-ms-identifiers"], []);
```

Comment:
This is not ignoring the values.  Ignoring the values would be not having an x-ms-identifiers extension.  If we want no identifiers in this case, we should use the decorstor override.